spec/scorecardSpec.js

```diff
@@ -0,0 +1,189 @@
+describe('Scorecard', function() {
+
+  var scorecard;
+
+  beforeEach(function() {
+    scorecard = new Scorecard(Frame);
+  });
+
+  describe('when initialised', function() {
+
+    it('has ten frames', function() {
+      expect(scorecard.frameRecord).toEqual([new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame, new Frame]);
+    });
+
+  });
+
+  describe('when playing', function() {
```

Comment:
use a `context`.  Rule of thumb, if it starts 'when...' or 'if...' then it's a context!